proto/reaction.proto

```diff
@@ -681,12 +812,13 @@ message DateTime {
 
 message ReactionAnalysis {
   // TODO(ccoley): Solicit more feedback from experimentalists
+  // TODO: Consider adding 1H/13C/OTHER_NMR fields
   enum AnalysisType {
     UNSPECIFIED = 0;
     CUSTOM = 1;
     LC = 2;  // Liquid chromatography, including HPLC and UPLC.
     GC = 3;  // Gas chromatography.
-    IR = 4;  // Infrared spectroscopy.
+    IR = 4;  // Infrared spectroscopy, including ReactIR.
     NMR = 5;  // NMR spectroscopy.
```

Comment:
We discussed replacing `NMR` with `NMR_1H`, `NMR_13C`, and `NMR_OTHER` here since these are _so_ common. Thoughts? The comment for `NMR_OTHER` could say "specify details in `details`"

Reply:
Agreed. Made the change and deleted the `TODO`.